cachi2/core/config.py

```diff
@@ -22,6 +25,18 @@ class Config(BaseModel, extra="forbid"):
     requests_timeout: int = 300
     concurrency_limit: int = 5
 
+    @model_validator(mode="before")
+    @classmethod
+    def _print_deprecation_warning(cls, data: Any) -> Any:
+        if "gomod_strict_vendor" in data:
+            log.warning(
+                "The `gomod_strict_vendor` config option is deprecated and will be removed in "
+                "future versions. Notice that it no longer has any effect when set, Cachi2 will "
```

Comment:
s/Notice/Note

Reply:
With the changes, the phrase no longer exists.

Reply:
I can still see it though :) .

Reply:
Right, this was on the deprecation warnings. Now it is fixed.